bears/general/CPDBear.py

```diff
@@ -0,0 +1,45 @@
+import re
+
+from coalib.bearlib.abstractions.Lint import Lint
+from coalib.bears.LocalBear import LocalBear
+
+
+class CPDBear(LocalBear, Lint):
+    executable = 'run.sh'
+    arguments = "cpd --minimum-tokens 10" + \
+                " --files {filename}"
+    output_regex = re.compile(
+        r'(?P<message>.*) in')
+
+    def run(self, filename, file):
+        '''
+        Checks duplicate code with ``cpd``.
+        Languages: C++, Java, C#, JavaScript,
+        Fortran, Go, JSP, Objective C,
+        PHP, Python, Ruby, Scala
+        '''
+        if filename.endswith('.cpp'):
```

Comment:
Makes a lot more sense to make this into a dictionary ? And use http://stackoverflow.com/questions/541390/extracting-extension-from-filename-in-python to get extensions ?

Like:
```
language_args = {
    "js": "ecmascript",
    "f": "fortran",
    "m": "objectivec",
    "rb": "ruby"}

_, extension = os.path.splitext(filename)
extension = extension[1:]  # Remove starting period
self.arguments += " --language {}".format(language_args.get(extension, extension)) 
```
And let the lint handle if the user inputs an invalid language ?

Seeems more generic